jwst/outlier_detection/utils.py

```diff
@@ -214,22 +292,31 @@ def flag_resampled_model_crs(
     else:
         pix_ratio = 1.0
 
-    blot = gwcs_blot(median_data, median_wcs, input_model.data.shape, input_model.meta.wcs, pix_ratio)
+    blot = gwcs_blot(median_data, median_wcs, input_model.data.shape,
+                     input_model.meta.wcs, pix_ratio, fillval=np.nan)
```

Comment:
same question as above about setting fillval to NaN; does this enforce NaN setting related to a different issue?